data/scoreboard/team.py

```diff
@@ -1,8 +1,32 @@
+import debug
+
+
 class Team:
-    def __init__(self, abbrev, runs, name, hits, errors, record):
+    def __init__(self, abbrev, runs, name, hits, errors, record, special_uniform):
         self.abbrev = abbrev
         self.runs = runs
         self.name = name
         self.hits = hits
         self.errors = errors
         self.record = record
+        self.special_uniform = special_uniform
+
+
+    def lookup_color(self, team_colors):
+        """
+        Given a team color object, this returns the colors for the team.
+
+        The result is guaranteed to have the 'home', 'accent', and 'text' keys.
+        Any missing data is filled in with the default colors.
+        If the team has a special uniform, the colors for that uniform are substuted
+        """
+        default_colors = team_colors.color("default")
```

Comment:
Does this also need to be behind a config?

Reply:
Not sure what you mean - there is a “default” section of the config that can be changed already

Reply:
I meant config to disable special uniform colors altogether

Reply:
Oh, I don’t think that’s necessary, because you could just delete those sub-keys or set them equal to the normal team colors and that would effectively disable it as well